gum/backend-darwin/gumdarwinmodule.c

```diff
@@ -542,21 +537,7 @@ gum_emit_import (const GumDarwinBindDetails * details,
     d.slot = 0;
   }
 
-  key = g_strconcat (
```

Comment:
We should also remove the local variable.

Reply:
oops